src/style/themes/small/small-theme.config.js

```diff
@@ -9,7 +9,8 @@ export default (palette) => {
         base: palette.productGreen,
         primary: palette.productGreenShade(21),
         secondary: palette.productGreenShade(41),
-        tertiary: palette.productGreenShade(61)
+        tertiary: palette.productGreenShade(61),
+        disabled: palette.productGreenTint(40)
```

Comment:
Shouldn't it be in disabled section to override disabled in baseTheme?

Reply:
They're used slightly differently, e.g. in the `switch-slider.style.js` I use both this and the baseTheme `disabled.disabled`

Reply:
Similarly, I couldn't override `disabled.disabled` from baseTheme, as this is being used in other components.